### PR TITLE
fix: renamed attribute should remain enumerable

### PR DESCRIPTION
**Pull request by cyjake**
even if there is getter method with same name. Also added `Model.Instance` in sequelize adapter

### CODE REVIEW

File: src/adapters/sequelize.js [hunk 1]
```diff
@@ -126,6 +126,10 @@ module.exports = Bone => {
       return true;
     }
 
+    static get Instance() {
+      return this;
+    }
```
Comment (cyjake) [inline src/adapters/sequelize.js:131]: added `Model.Instance` alias

File: test/unit/bone.test.js [hunk 2]
```diff
@@ -5,119 +5,178 @@ const { Bone, DataTypes, connect } = require('../..');
...
-      { columnName: 'ssn', columnType: 'varchar', dataType: 'varchar', primaryKey: true },
-    ]);
-    assert.equal(User.primaryKey, 'ssn');
-    assert.equal(User.primaryColumn, 'ssn');
+      User.load([
+        { columnName: 'foo', columnType: 'varchar', dataType: 'varchar' },
+      ]);
+      User.renameAttribute('foo', 'bar');
+      assert.ok(Object.getOwnPropertyDescriptor(User.prototype, 'bar').enumerable);
+    });
```
Comment (cyjake) [inline test/unit/bone.test.js:159]: It's expected that attributes should always present in `toJSON()` or `toObject()` results. Make renamed attribute enumerable as well.

File: src/bone.js [hunk 3]
```diff
@@ -936,23 +936,7 @@ class Bone {
     }
 
     const descriptors = {};
```
Comment (JimmyDaddy) [inline src/bone.js:938]:
```suggestion
```

File: src/bone.js [hunk 4]
```diff
@@ -936,23 +936,7 @@ class Bone {
...
-      }, Object.keys(descriptor || {}).reduce((result, key) => {
-        if (descriptor[key] != null) result[key] = descriptor[key];
-        return result;
-      }, {}), {
-        enumerable: true,
-        configurable: true,
-      });
-    }
+    for (const name in attributes) this.loadAttribute(name);
     Object.defineProperties(this.prototype, descriptors);
```
Comment (JimmyDaddy) [inline src/bone.js:940]:
```suggestion
```
unused code

Comment (cyjake) [reply src/bone.js]: nice catch

**Review by JimmyDaddy — APPROVED** [review]

**Pull request merged**